docs/CLI/gateway.md

```diff
@@ -29,7 +29,7 @@ To create a gateway using the CLI you will need to run the following command:
 The next step will be to configure the custom domain for your gateway. 
 
 ```shellscript filename="Configuring the custom domain" copy
-> fleek gateway domain add
+> fleek gateway create
```

Comment:
The edits look great! 
one thing i noticed in the docs was that it should be `gateways` instead of `gateway` at all places